front/tests/pages/stdcm-page-model.ts

```diff
@@ -230,6 +250,16 @@ class STDCMPage {
     await hourLocator.click();
   }
 
+  private getSimulationLengthAndDurationLocator(simulationNumber: number): Locator {
+    return this.simulationList
+      .locator('.simulation-metadata .total-length-trip-duration')
+      .nth(simulationNumber - 1);
+  }
+
+  private getSimulationNameLocator(simulationNumber: number): Locator {
+    return this.simulationList.locator('.simulation-name').nth(simulationNumber - 1);
```

Comment:
We can't use `last()` because this function is intended to test two simulations. It would be more useful when we enhance the test by adding functionality to navigate between simulations